tutorials/create-esxi-cluster-dedibox/index.mdx

```diff
@@ -324,45 +320,45 @@ To proceed with these steps, you require at least one [failover IP](https://docu
 
-18. Double click the downloaded ISO file to display it as a virtual CD-ROM drive in Windows. Double click the `autorun.exe` application to launch the installation wizard:
+17. Double click the downloaded ISO file to display it as a virtual CD-ROM drive in Windows. Double click the `autorun.exe` application to launch the installation wizard:
 
   <Lightbox src="scaleway-autorun.png" width="500" alt="" />
 
-19. During the setup wizard, choose the embedded setup, which allows you to install all vCenter components on a single machine:
+18. During the setup wizard, choose the embedded setup, which allows you to install all vCenter components on a single machine:
 
   <Lightbox src="scaleway-vcenter_embedded.png" width="500" alt="" />
 
-20. Choose a system network name for your vCenter. This name must be a working FQDN:
+19. Choose a system network name for your vCenter. This name must be a working FQDN:
 
   <Lightbox src="scaleway-vcenter_name.png" width="500" alt="" />
 
-21. Configure the credentials for the single sign-on system of vCenter:
+20. Configure the credentials for the single sign-on system of vCenter:
 
   <Lightbox src="scaleway-vcenter_signon.png" width="500" alt="" />
 
-22. Set the vCenter service account. You can keep the default settings and continue by clicking **Next**:
+21. Set the vCenter service account. You can keep the default settings and continue by clicking **Next**:
 
   <Lightbox src="scaleway-vcenter_service_account.png" width="500" alt="" />
 
-23. Configure the vCenter database. You can keep the default settings so it uses an internal Postgres database:
+22. Configure the vCenter database. You can keep the default settings so it uses an internal Postgres database:
 
   <Lightbox src="scaleway-vcenter_database.png" width="500" alt="" />
 
-24. The default ports used by the vCenter application are displayed. You can keep the default settings, and click **Next** to continue:
+23. The default ports used by the vCenter application are displayed. You can keep the default settings, and click **Next** to continue:
 
   <Lightbox src="scaleway-vcenter_ports.png" width="500" alt="" />
 
-25. Decide if you want to participate in the "Customer Experience Improvement Program" by sharing data with VMware. Untick the box if not, and click **Next** to continue
+24. Decide if you want to participate in the "Customer Experience Improvement Program" by sharing data with VMware. Untick the box if not, and click **Next** to continue
 
   <Lightbox src="scaleway-vcenter_cei.png" width="500" alt="" />
 
-26. A summary of your configuration displays. Check again if all your settings are correct and click **Install** to launch the installation of VMware vCenter
+25. A summary of your configuration displays. Check again if all your settings are correct and click **Install** to launch the installation of VMware vCenter
```

Comment:
Put " A summary of your configuration displays" as a seperate line (no number) after the image. Start this step with "Check again..."